src/base/vec_storage.rs

```diff
@@ -281,6 +281,12 @@ where
     {
         self.clone()
     }
+
+    #[inline]
+    fn forget(mut self) {
+        // Set length to 0 so element destructors are not called.
```

Comment:
Perhaps add a "SAFETY" comment here and explain that `.set_len(0)` always satisfies the invariants of `.set_len`, but the contents are deliberately leaked

(same for other `forget` impl)

the "SAFETY comment" practice is a relatively recent thing, and much of `nalgebra`'s codebase, to my knowledge, predates this convention. But I think we should try to stick with it going forward, whenever it makes sense!